### DISCUSSION TITLE
Troubleshooting: Commits Not Showing on Overview Page for Last 15 Days

### DISCUSSION THREAD

**Discussion by Bhupi2508** [community/community#57142] [Actions]
### Select Topic Area

Question

### Body

![Commits](https://github.com/community/community/assets/47639636/726e3053-c3fa-489c-9829-9c0058b75c46)
I created a private repository and pushed around 30 commits within the last 15-16 days. However, I cannot see those commits in green color on the overview page. I changed the repository's visibility to public, but I still can't see the commits or the latest ones on the overview page. Can you please assist me with this issue?

**Comment by radar-blip** [comment]
Maybe this will help you, its in your settings.

![Screenshot from 2023-06-04 02-59-31](https://github.com/community/community/assets/39196767/861a1e86-125e-451b-a7b2-299698afef2a)

**Reply by Bhupi2508** [reply]
Thank you for the quick response, but I have already completed both tasks. However, I am still unable to see the desired result. I am unsure of the reason for this.
![github](https://github.com/community/community/assets/47639636/f58ef25b-b28e-4aa6-b4f0-c04b054b5b72)
